audiodecoder.asap/addon.xml.in

```diff
@@ -6,12 +6,13 @@
   provider-name="spiff">
   <requires>@ADDON_DEPENDS@</requires>
   <extension
+    language="C++"
```

Comment:
Isn't the effect of this PR to change the language from C++ to C?

Reply:
Ah yeah :smile:, has forget to change. Think about "C" and "C++" not so complicated :sweat_smile: , thought to use this for values like "js", "python4", "java" and to know how to start.